starters/next-drupal-starter/lib/getPreview.js

```diff
@@ -54,8 +54,7 @@ export async function getPreview(context, node) {
   if (context?.previewData?.resourceVersionId) {
     process.env.DEBUG_MODE &&
       console.log(
-        `Adding recource version ID param ${
-        context?.previewData?.resourceVersionId}...`
+        `Adding resource version ID param ${context?.previewData?.resourceVersionId}...`
       );
 
     store.params.addCustomParam({
```

Comment:
Ah whoops!

I updated `getPreview` to return an updated params string so we can run the subsequent `getObject` with the new params in case of a revision. I was able to get that mostly working but I'm running into some strange behavior where when fetching a revision, I am getting the data for the original content back, not the revision. Clearing the preview data and/or the cache in Drupal does not change this behavior. Any ideas?

Reply:
I am realizing this is because even with the DrupalState `refresh` param, we are getting stale date from the store. This behavior will need that update to DrupalState that invalidates the local store if the params change. Will push up the current work for this.